.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname $0)/_/husky.sh"
```

Comment:
what is purpose of it?

Reply:
Generated by husky, script executing husky.sh

Reply:
actually whole file is a pre-commit file, i mean it's also named pre-commit maybe it should be named prettier-stage?